firmware/user/modes/mode_color_movement.c

```diff
@@ -760,7 +761,23 @@ void ICACHE_FLASH_ATTR cmGameInput(void)
         cmNewSetup(cmCurrentSubMode);
         cmChangeState(CM_GAME);
     }
-    //button a = cycle brightness
+    //button a previous submode
```

Comment:
Please revert the bidirectional scrolling.

Reply:
Sadface, reverted.